js/ui/popup/factories/httpsTester.js

```diff
@@ -0,0 +1,42 @@
+(function () {
+    'use strict';
+
+    /**
+     * @ngdoc function
+     * @name passmanApp.controller:MainCtrl
+     * @description
+     * # MainCtrl
+     * Controller of the passmanApp
+     */
+    angular.module('passmanExtension').factory('HttpsTest', ['$http', '$q', function ($http, $q) {
+        var tester = {};
+        tester.test = function (url) {
+            var deferred = $q.defer();
+            if(url.match(/https?/)){
```

Comment:
make the regex /^http?/ in case url contains http somewhere else than the beginning